app/models/track.rb

```diff
@@ -0,0 +1,3 @@
+class Track < ApplicationRecord
+  has_many :track_units
```

Comment:
Whats the intention with this `TrackUnit` model? it seems to replicate courses going by the dummy tracks file below. Can we not make it so the `Track` model has many courses instead?

Reply:
> By no means is this code good/done, just getting the ball rolling here

No worries, I completely understand that.

> The courses is too large of a unit of study.

Can you explain this a bit further please? I thought that a track would be made up of multiple courses.

> We want the ability to choose at a lesson-level which lessons belong to a track-unit (Read: "Module") and which track_units belong to a track

I reckon we will want to do it from a section level. There aren't any instances where we would one lesson from a section and not the others as they are all designed to be taken together. At the moment sections belong to specific courses but we could change that so they could belong to many courses.